internal/config/canary_config.go

```diff
@@ -19,101 +19,109 @@ import (
 
 const (
 	// environment variables declaration
-	BootstrapServersEnvVar            = "KAFKA_BOOTSTRAP_SERVERS"
-	BootstrapBackoffMaxAttemptsEnvVar = "KAFKA_BOOTSTRAP_BACKOFF_MAX_ATTEMPTS"
-	BootstrapBackoffScaleEnvVar       = "KAFKA_BOOTSTRAP_BACKOFF_SCALE"
-	TopicEnvVar                       = "TOPIC"
-	ReconcileIntervalEnvVar           = "RECONCILE_INTERVAL_MS"
-	ClientIDEnvVar                    = "CLIENT_ID"
-	ConsumerGroupIDEnvVar             = "CONSUMER_GROUP_ID"
-	ProducerLatencyBucketsEnvVar      = "PRODUCER_LATENCY_BUCKETS"
-	EndToEndLatencyBucketsEnvVar      = "ENDTOEND_LATENCY_BUCKETS"
-	ExpectedClusterSizeEnvVar         = "EXPECTED_CLUSTER_SIZE"
-	KafkaVersionEnvVar                = "KAFKA_VERSION"
-	SaramaLogEnabledEnvVar            = "SARAMA_LOG_ENABLED"
-	VerbosityLogLevelEnvVar           = "VERBOSITY_LOG_LEVEL"
-	TLSEnabledEnvVar                  = "TLS_ENABLED"
-	TLSCACertEnvVar                   = "TLS_CA_CERT"
-	TLSClientCertEnvVar               = "TLS_CLIENT_CERT"
-	TLSClientKeyEnvVar                = "TLS_CLIENT_KEY"
-	TLSInsecureSkipVerifyEnvVar       = "TLS_INSECURE_SKIP_VERIFY"
-	SASLMechanismEnvVar               = "SASL_MECHANISM"
-	SASLUserEnvVar                    = "SASL_USER"
-	SASLPasswordEnvVar                = "SASL_PASSWORD"
+	BootstrapServersEnvVar              = "KAFKA_BOOTSTRAP_SERVERS"
```

Comment:
Should we really use this type of formatting? i'm not sure if it makes the code readable for someone - maybe it does. But it definitely makes PR reviews, diffs and so on much more complicated.

Reply:
I totally agree with you @scholzj but I am just following the Golang best practices where everything is aligned in this way and the tooling you have in the IDE does it for you automatically. This comes from old C ... you know ;-)

Reply:
Should we follow practices which suck? :-o

Anyway, as I said, something to think about ... not really something to change in this PR.

Reply:
in my opinion it sucks from a PR review point of view and I totally agree with that, but I like this alignment ... maybe because I developed in C a lot in the past? :-)